testassets/InteropTestsClient/RunGrpcTests.ps1

```diff
@@ -31,21 +33,15 @@ $allTests =
 
 Write-Host "Running $($allTests.Count) tests" -ForegroundColor Cyan
 Write-Host "Use TLS: $use_tls" -ForegroundColor Cyan
+Write-Host "Use WinHttp: $use_winhttp" -ForegroundColor Cyan
+Write-Host "Framework: $framework" -ForegroundColor Cyan
 Write-Host
 
 foreach ($test in $allTests)
 {
   Write-Host "Running $test" -ForegroundColor Cyan
 
-  if (!$use_tls)
-  {
-    dotnet run --use_tls false --server_port 50052 --client_type httpclient --test_case $test
-  }
-  else
-  {
-    # Certificate is for test.google.com host. To run locally, setup the host file to point test.google.com to 127.0.0.1
```

Comment:
isn't it dangerous to drop tests with --use_tls=true?

Reply:
Using a cert with test.google.com never successfully worked so I removed the if/else.

This file is just a helper for running all the interop tests locally and isn't used by CI servers.